ex3/ex3.c

```diff
@@ -9,7 +9,12 @@
 
 int main(void)
 {
-    // Your code here
-
+    fork();
+    if (fork() == 0){
```

Comment:
If you run this program, you'll see an output:
```
hello
hello
goodbye
goodbye
```
Think the spec should be 1 "hello" and 1 "goodbye" in that order, but it's doubled in yours. Why would that be?
(Hint: _Look at your usage of `fork()`_)